### DISCUSSION TITLE
Using spectrum classes/tokens

### DISCUSSION THREAD

**Discussion by conormdowney** [adobe/react-spectrum#2094] [General]
So Im trying to use the field label as per: https://opensource.adobe.com/spectrum-css/fieldlabel.html

I have the following code, this is a React project: 

```
<div>
            <span className={props.insight.type}></span>
            <label className="spectrum-FieldLabel spectrum-FieldLabel--sizeM">{componentNameMap[props.insight.type]}</label>
            <ButtonGroup align="end">
                <Button variant="cta" onPress={sleep}>Sleep</Button>
                <Button variant="cta" onPress={dismiss}>Dismiss</Button>
            </ButtonGroup>
        </div>
```

but when I run my project i dont have the styling on the label.  When I inspect i dont see the classes in the styles tab. I have run npm install @spectrum-css/fieldlabel and I see it in my node modules and package.json.

I assume I have to import them into my project somehow but im not sure how.

**Comment by snowystinger** [comment]
Are you importing the CSS? With most bundlers, you can import it as you would code. Sometimes you need to specify a loader for the CSS as well.

**Reply by snowystinger** [reply]
One thing to note, you'll end up with two copies of the FieldLabel CSS as we use a CSS Module version that will be different than yours. Is there a reason why you need a label that isn't attached to or labeling anything?
